instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/context/ParentPropagatingContext.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.context;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.ContextKey;
+import io.opentelemetry.context.Scope;
+
+public class ParentPropagatingContext implements Context {
```

Comment:
this is what tracers can return from `startOperation` if they want to suppress the span